api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -62,7 +60,22 @@ type IBMVPCMachineSpec struct {
 	PrimaryNetworkInterface NetworkInterface `json:"primaryNetworkInterface,omitempty"`
 
 	// SSHKeys is the SSH pub keys that will be used to access VM.
-	SSHKeys []*string `json:"sshKeys,omitempty"`
+	SSHKeys []*IBMVPCResourceReference `json:"sshKeys,omitempty"`
```

Comment:
what will happen when user supply both ID and Name? can we add that information?

Reply:
Added note.